velox/experimental/gpu/tests/HashTableTest.cu

```diff
@@ -280,25 +280,26 @@ __global__ void probe<true>(
     auto rem = hash % sizeof(uint32_t);
     int64_t j = hash - rem;
     uint32_t cmpMask = 0xffffffff << (rem * 8);
-    for (;;) {
-      auto hits = __vcmpeq4(*(uint32_t*)&table->tags[j], tag) & cmpMask;
-      while (hits) {
-        auto jj = j + (__ffs(hits) - 1) / 8;
-        if (table->keys[jj] == keys[i]) {
-          hasValue[i] = true;
-          values[i] = table->values[jj];
-          goto end;
+    ([&](){
```

Comment:
Not sure if this version is more readable.  Although it's just test code.

Reply:
That is to fix a warning from the compiler that complains of the `end:` right before the for loop's end `}`. The warning is something like:
```
velox/experimental/gpu/tests/HashTableTest.cu(653): warning #127-D: expected a statement
    }
    ^

Remark: The warnings can be suppressed with "-diag-suppress <warning-number>"
```

That's also not just a warning on some compilers which are more strict.

Reply:
Can we just add a `;` after `end:` to fix the warning?

Reply:
Yes, that fixes it! Thanks for the tip